presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -319,6 +365,7 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
         builder.add(new IterativeOptimizer(
                 stats,
                 statsCalculator,
+                estimatedExchangesCostCalculator,
```

Comment:
Maybe we could pass a dummy calculators by default (one which only throws unsupported exception)? To control who is accessing the stats and cost calculator.

Reply:
i don't understand, please explain